GrocifyApp.UI/src/pages/Products/ProductSections.jsx

```diff
@@ -48,12 +48,12 @@ function ProductSections() {
         };
 
         fetchData();
-    });
+    }, []);
```

Comment:
can you set token as a dependency? it gives a warning:

![image](https://github.com/Catarina98/Grocify/assets/36730010/4d88043f-e9f5-41ee-b56d-d75fe86a984f)